### ISSUE TITLE
Resume an experiment from dashboard

### ISSUE THREAD

**Issue by LiamHe** [mlbench/mlbench#65]
**Is your feature request related to a problem? Please describe.**
For the moment, we can't not resume an experiment by its `run_id`.

**Describe the solution you'd like**
Add a button next to `add run` and select an experiment to resume. If the selected experiment has no checkpoints, then rerun the experiment from the very beginning.

**Describe alternatives you've considered**
Add the resume button inside the `run` page for each experiment.

**Comment by Panaetius** [comment]
A stop/pause experiment button would also make sense for this usecase

**Comment by LiamHe** [comment]
By saying `pause/stop`, do you mean we kill the mpi job? And resume from checkpoints later.

**Comment by Panaetius** [comment]
yes. since right now the only way to stop a job is by deleting the run.
